how-to/integrate-with-sso/README.md

```diff
@@ -0,0 +1,83 @@
+<img src="../../assets/OpenFin-Workspace-Starter.png" width="100%" alt="OpenFin Workspace Example Application -- Integrate with SSO" />
+
+> **_:information_source: OpenFin Workspace:_** [OpenFin Workspace](https://www.openfin.co/workspace/) is a commercial product and this repo is for evaluation purposes (See [LICENSE.MD](LICENSE.MD)). Use of the OpenFin Container and OpenFin Workspace components is only granted pursuant to a license from OpenFin (see [manifest](public/manifest.fin.json)). Please [**contact us**](https://www.openfin.co/workspace/poc/) if you would like to request a developer evaluation key or to discuss a production license.
+> OpenFin Workspace is currently **only supported on Windows** although you can run the sample on a Mac for development purposes.
+
+# Integrate with SSO
+
+Your OpenFin application will often need to authenticate with a SSO provider before use, this examples demonstrates such an integration.
+
+This application you are about to install is a simple example of plugging in your own content or app. This example assumes you have already [set up your development environment](https://developers.openfin.co/of-docs/docs/set-up-your-dev-environment)
```

Comment:
This example doesn't really show that does it? Do we need that first sentence?

Reply:
We don't mention anywhere else what the sample does